nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -420,6 +120,33 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 		warning_cooldown = TRUE
 		addtimer(VARSET_CALLBACK(src, warning_cooldown, FALSE), 5 SECONDS)
 		return
+	if(damage_amount > max_integrity/2) //Taking a single hit that does over 50% of your max HP is BAD
+		var/destroyed = pick(1, 2, 3, 4) //May as well go AWOL at this point, MAA is going to lynch you
+		for(var/I = 0, I < destroyed, I++)
+			burnout_component()
+		return
+	else if(damage_amount >= 50)

```

Comment:
Should these scale with max as well or nah?

Reply:
If they scaled with max, they would likely exceed the damage bounds of the intended impact, but definitely something to look at in P3.